commands/elevatorsetting.py

```diff
@@ -76,3 +79,79 @@ def __init__(self, elevatorSubsystem: ElevatorSubsystem) -> None:
 
     def execute(self) -> None:
         self.elevator.setIntakePosition()
+
+
+class ElevatorManualMode(SetElevatorState):
+    def __init__(self, elevatorSubsystem: ElevatorSubsystem) -> None:
+        SetElevatorState.__init__(self, elevatorSubsystem)
+
+    def execute(self) -> None:
+        self.elevator.setManualMode()
+
+
+class ElevatorManualUp(Command):
+    def __init__(self, elevator: ElevatorSubsystem) -> None:
+        Command.__init__(self)
+        self.setName(__class__.__name__)
+        self.elevator = elevator
+
+        self.elevatorPositionPublisher = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kElevatorPositionKey)
+            .publish()
+        )
+        self.elevatorPositionGetter = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kElevatorPositionKey)
+            .subscribe(constants.kIntakePositionBeltPosition)
+        )
+        self.elevatorStateGetter = (
+            NetworkTableInstance.getDefault()
+            .getStringTopic(constants.kElevatorStateKey)
+            .subscribe("ElevatorState.IntakePosition")
+        )
+
+    def execute(self):
+        if self.elevatorStateGetter.get() == "ElevatorState.ManualMode":
```

Comment:
you already require the elevator subsystem, you shouldn't need to get elevator state from nettables